sitzverteilung-frontend/src/components/basedata/BaseDataForm.vue

```diff
@@ -147,12 +147,11 @@ function validChanged(valid: boolean | null) {
 }
 
 const seatFieldValidationError = computed(() => {
+  if (isTooManyGroups.value && expectedSeats.value > 0) {
+    return `Die Anzahl an Parteien / Gruppierungen / Einzelmitglieder übersteigt den ${expectedSeats.value > groups.value.length ? "maximalen" : "angegebenen/maximalen"} Wert.`;
+  }
```

Comment:
"Maximal" kann doch jetzt nur noch auftreten, wenn ich eine Hauptorgangröße eingebe? Das ist ja auch doof,
Zudem bezieht sich maximal jetzt nicht mehr auf auf das definierte  Limit?


```suggestion
  if (isTooManyGroups.value) {
    return `Die Anzahl an Parteien / Gruppierungen / Einzelmitglieder übersteigt den ${expectedSeats.value > 0 ? "angegebenen / maximalen" : "maximalen"} Wert.`;
  }
```